ui/app/services.js

```diff
@@ -208,9 +208,6 @@
             case 404:
               Notifications.error('Requested resource not found');
               break;
-            case 409:
```

Comment:
After removal of this case branch, will the proper error message show up?

I'm all hands for it. My only concern is that it may result in appropriate messages not showing up if backend returns something that UI is not able to parse and display. And I don't see any backend changes here.